applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_dynamic_U_Pw_scheme.hpp

```diff
@@ -289,6 +289,30 @@ class NewmarkDynamicUPwScheme : public NewmarkQuasistaticUPwScheme<TSparseSpace,
         KRATOS_CATCH( "" )
     }
 
+    void CalculateRHSContribution(
+        Condition& rCurrentCondition,
+        LocalSystemVectorType& RHS_Contribution,
+        Element::EquationIdVectorType& EquationId,
+        const ProcessInfo& CurrentProcessInfo) override
+    {
+        KRATOS_TRY
+
+        int thread = OpenMPUtils::ThisThread();
+
+        (rCurrentCondition).CalculateRightHandSide(RHS_Contribution, CurrentProcessInfo);
+
+        (rCurrentCondition).CalculateMassMatrix(mMassMatrix[thread], CurrentProcessInfo);
+
+        (rCurrentCondition).CalculateDampingMatrix(mDampingMatrix[thread],CurrentProcessInfo);
```

Comment:
I believe there is no need to enclose `rCurrentCondition` in parentheses here. If you agree, I would suggest to remove them. Thanks.